bedrock/firefox/views.py

```diff
@@ -382,8 +382,11 @@ def get_template_names(self):
         if detect_channel(version) == 'alpha':
             template = 'firefox/dev-firstrun.html'
         elif show_40_firstrun(version):
+            #New onboarding animation funnecake
```

Comment:
Looks like this is causing the flake 8 error: ``bedrock/firefox/views.py:385:13: E265 block comment should start with '# '``